helm/oauth2-proxy/templates/deployment.yaml

```diff
@@ -172,6 +172,11 @@ spec:
         lifecycle:
 {{ toYaml .Values.lifecycle | indent 10 }}
 {{- end }}
+        {{- with .Values.proxyVarsFromSecret }}
+        envFrom:
+        - secretRef:
+            name: {{  . }}
+        {{- end }}
         env:
         {{- if .Values.proxyVarsAsSecrets }}
```

Comment:
proxyVarsAsSecrets and proxyVarsFromSecret should be mutually exclusive. Therefore I would like to see this check extend that proxyVarsAsSecrets aren't rendered with proxyVarsFromSecret is set

Reply:
This PR was given to us as a "gift"; if we want to bring it in, we must introduce the changes we want. 🤷🏻‍♂️